src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ReleasePermissionService.cs

```diff
@@ -0,0 +1,177 @@
+#nullable enable
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Admin.Services.Interfaces;
+using GovUk.Education.ExploreEducationStatistics.Admin.Services.Interfaces.Security;
+using GovUk.Education.ExploreEducationStatistics.Admin.ViewModels;
+using GovUk.Education.ExploreEducationStatistics.Common.Model;
+using GovUk.Education.ExploreEducationStatistics.Common.Services.Interfaces.Security;
+using GovUk.Education.ExploreEducationStatistics.Common.Utils;
+using GovUk.Education.ExploreEducationStatistics.Content.Model;
+using GovUk.Education.ExploreEducationStatistics.Content.Model.Database;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using static GovUk.Education.ExploreEducationStatistics.Content.Model.ReleaseRole;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Services
+{
+    public class ReleasePermissionService : IReleasePermissionService
+    {
+        private readonly ContentDbContext _contentDbContext;
+        private readonly IPersistenceHelper<ContentDbContext> _persistenceHelper;
+        private readonly IUserReleaseRoleRepository _userReleaseRoleRepository;
+        private readonly IUserService _userService;
+
+        public ReleasePermissionService(
+            ContentDbContext contentDbContext,
+            IPersistenceHelper<ContentDbContext> persistenceHelper,
+            IUserReleaseRoleRepository userReleaseRoleRepository,
+            IUserService userService)
+        {
+            _contentDbContext = contentDbContext;
+            _persistenceHelper = persistenceHelper;
+            _userReleaseRoleRepository = userReleaseRoleRepository;
+            _userService = userService;
+        }
+
+        public async Task<Either<ActionResult, List<ContributorViewModel>>>
+            ListReleaseContributors(Guid publicationId, Guid releaseId)
```

Comment:
We should also remove the `publicationId` as argument for this method, and also remove it from the controller endpoint as well